### PR TITLE
bump dependencies

### LINKED ISSUE #97: Invalid number with valid format in IOS
When trying to check phone number validation it returns invalid even in a valid format. It's having on IOS devices only and not on all IOS devices. I saw that the native package you are using is updated recently but you haven't updated the package version in your package. Hopefully, you will solve the issue as soon as possible.

### PR DESCRIPTION
**Pull request by VictorUvarov**
## Connection with issue(s)

Close #97 

Connected to #92

## Solution description

Based on the discussion in #97. The underlying iOS phone number package (PhoneNumberKit) has stopped supporting cocoa pods installation via cocoapods.org and instead has moved over to asking users to just use the pod via a git URL. [Details](https://github.com/marmelroy/PhoneNumberKit#setting-up-with-cocoapods)

The Cocoapods [spec](https://guides.cocoapods.org/syntax/podspec.html) doesn't support using git URLs in the pod spec file directly, as far as I understand. We currently have to use the git link in the project's Podfile. This requires the user to do 1 setup step in their project after installing the dependency and updating 2 files when updating the phone_number dependency in their `pubspec.yaml` file.

## To Do

- [x] Read [contributing guide](https://github.com/flutter-form-builder-ecosystem/.github/blob/main/CONTRIBUTING.md)
- [x] Check the original issue to confirm it is fully satisfied
- [x] Add solution description to help guide reviewers
- [ ] Add unit test to verify new or fixed behaviour
- [x] If apply, add documentation to code properties and package readme

### CODE REVIEW

**Comment by deandreamatias** [comment]
Really thanks for this contribution. I will merge when pass pipeline and deploy a new version to pub.dev

File: README.md [hunk 1]
```diff
@@ -43,14 +43,20 @@ The plugin uses the native libraries [libphonenumber](https://github.com/google/
...
 ### Setup
 
-No especific setup required: only install the dependency and use :)
+Add the following line to the top of the `Runner` target in your `ios/Podfile`
+```ruby
+target 'Runner' do
+  pod 'PhoneNumberKit/PhoneNumberKitCore', :git => 'https://github.com/marmelroy/PhoneNumberKit', :tag => '3.5.8'
+  ...
+end
+```
```
Comment (deandreamatias) [inline README.md:59]: I think that need add this on Podfile example, right?

Comment (VictorUvarov) [reply README.md:59]: The example Podfile does have the change

**Comment by hasibul-hasan-shuvo** [comment]
@deandreamatias When the updates will be reflected in the package?

**Comment by deandreamatias** [comment]
> @deandreamatias When the updates will be reflected in the package?

Hi @hasibul-hasan-shuvo 
I will try release new versions in the next days. I'm working on [form_builder_file_picker](https://github.com/flutter-form-builder-ecosystem/form_builder_file_picker) right now

**Comment by hasibul-hasan-shuvo** [comment]
Hi @deandreamatias, Actually I'm having this issue in a live app so that's an emergency for me. It would be great if you can push the updates as soon as possible. Thank you.

**Comment by deandreamatias** [comment]
if is urgent, please use package directly from this repo @hasibul-hasan-shuvo 
https://dart.dev/tools/pub/dependencies#git-packages

**Comment by markglibres** [comment]
same here.. this is a problem for newer iOS versions and we also need it to fix an issue on prod

**Pull request merged**